### DISCUSSION TITLE
Game Day Exterior Lighting Automation

### DISCUSSION THREAD

**Discussion by DrewGilbey** [vasqued2/ha-teamtracker#129] [Q&A]
I'm trying to setup a game day lighting automation.

I believe I can setup trigger and sunrise and use a condition to dictate if the lights turn on team colors.

Can I accomplish this with the attribute set to Date and what value do I need in State* to accomplish this?

**Comment by vasqued2** [comment]
The state would be PRE, but I don't think you need to worry about state.  Just check and see if the date in the attribute `date` is the same as today's date and you should be good.  Keep in mind that the date attribute is GMT so you might have to localize it to your time zone first.

Make sense?

**Reply by DrewGilbey** [reply]
Makes sense, thank you
